edx_proctoring/templates/proctoring/seq_proctored_exam_entrance.html

```diff
@@ -0,0 +1,84 @@
+{% load i18n %}
+<div class="sequence proctored-exam entrance" data-exam-id="{{exam_id}}">
+  <h3>
+    {% blocktrans %}
+      Would you like to take {{ display_name }} as a proctored exam?
+    {% endblocktrans %}
+  </h3>
+  <p>
+    {% blocktrans %}
+      Since you're enrolled in this course as a verified student, you have the option to take this exam with
+      online proctoring. Online proctoring is one requirement towards being eligible for credit.
+    {% endblocktrans %}
+  </p>
+  <div class="gated-sequence">
+    <span><i class="fa fa-lock"></i></span>
+    <a class="start-timed-exam" data-ajax-url="{{enter_exam_endpoint}}" data-exam-id="{{exam_id}}" data-attempt-proctored=true data-start-immediately=false>
+      {% trans "Yes, I want to take this exam with online proctoring (and be eligible for credit)" %}
+    </a>
+    <p>
+      {% blocktrans %}
+        You will be guided through installing {{platform_name}} approved online proctoring software and
```

Comment:
"You will be guided through steps to set up online proctoring software and to perform various checks. Have your photo ID ready so that you can verify your identity."
"Be ready to start your exam after you complete the proctoring setup."
@griffresch ?